### DISCUSSION TITLE
browser_check plugin fails in v8

### DISCUSSION THREAD

**Discussion by kschuler** [jspsych/jsPsych#3397] [General]
browser_check no longer displays the `exclusion_message` on true, I think because of this change in v8: 

```
We've renamed jsPsych.endExperiment() to jsPsych.abortExperiment().

We've renamed jsPsych.endCurrentTimeline() to jsPsych.abortCurrentTimeline().

```

browser_check is still trying to call jsPsych.endExperiment()

**Comment by jadeddelta** [comment]
hi @kschuler, sorry for the late reply but if you're still able to, do you still have the code that replicates this bug? i've tried to create it using various `inclusion_function`s, but to no avail. it looks like `browser-check` no longer calls `endExperiment()`, but rather has a wrapped function `end_experiment()` which calls `abortExperiment()` at the very end.

and if anyone else is having a bug like this concerning `browser_check`, feel free to comment it below so that we can get to the bottom of this !
